src/ui-layout.js

```diff
@@ -239,14 +240,14 @@ angular.module('ui.layout', [])
         for(i=0; i < ctrl.containers.length; i++) {
           if(!LayoutContainer.isSplitbar(ctrl.containers[i])) {
 
-            var child = ctrl.containers[i].element;
+            var container = ctrl.containers[i],
+              child = container.element;
             opts.maxSizes[i] = child.attr('max-size') || child.attr('data-max-size') || opts.maxSizes[i] || null;
             opts.minSizes[i] = child.attr('min-size') || child.attr('data-min-size') || opts.minSizes[i] || null;
-            opts.sizes[i] = child.attr('size') || child.attr('data-size') || opts.sizes[i] || 'auto';
-            //opts.collapsed[i] = child.attr('collapsed') || opts.collapsed[i] || false;
-
 
+            opts.sizes[i] = child.attr('size') || child.attr('data-size') || opts.sizes[i];
```

Comment:
Because the next line takes care of it in case there's no size provided by the user. Am I missing something?